CITATION.cff

```diff
@@ -0,0 +1,13 @@
+# YAML 1.2
+---
+authors: 
+  - 
+    name: "The Manim Community Developers"
+cff-version: "1.1.0"
+date-released: 2021-08-02
+license: MIT
+message: "We acknowledge the importance of good software to support research, and we note that research becomes more valuable when it is communicated effectively. To demonstrate the value of Manim, we ask that you cite Manim in your work."
+title: Manim
```

Comment:
Quick question regarding the title: Just "Manim" vs. "Manim – Mathematical Animation Framework" (as it was in the bibtex snippet)?

Reply:
I've changed it to the second one.